WordPress/src/main/java/org/wordpress/android/ui/prefs/DeleteSiteDialogFragment.java

```diff
@@ -76,7 +83,7 @@ public void onClick(DialogInterface dialog, int which) {
             public void onClick(DialogInterface dialog, int which) {
                 Fragment target = getTargetFragment();
                 if (target != null) {
-                    target.onActivityResult(getTargetRequestCode(), Activity.RESULT_OK, null);
+                    target.onActivityResult(getTargetRequestCode(), Activity.RESULT_OK, new Intent());
```

Comment:
Do you think it would be better if we kept the data `null` and instead check for this outside of `if (data != null)` statement? I don't think we care about the data, and someone might accidentally remove the `new Intent()` since it "looks" kind of pointless here. It's not a big deal but it also uses extra memory (a tiny bit, that is :P )